src/main/java/org/broadinstitute/hellbender/tools/funcotator/DataSourceFuncotationFactory.java

```diff
@@ -135,6 +135,12 @@ public String getVersion() {
         return version;
     }
 
+    /**
+     * @return {@code True} if this {@link DataSourceFuncotationFactory} requires features to create {@link Funcotation}s.  {@code False} otherwise.
```

Comment:
Any features?  Or specific types?  Please update docs.

Reply:
Yes - any features.  If the DataSourceFuncotationFactory relies on feature matching that the GATK engine provides, then this should be true.

Things that don't require features (like gene name matching data sources) don't require any features to be present to create a funcotation.